.github/workflows/dev-test.yml

```diff
@@ -0,0 +1,36 @@
+name: Dev_Test
+
+on: [pull_request]
```

Comment:
maybe on `[push]`?

I'm runnuing out of ideas.. maybe we can merge it and see what happens.